examples/bri-3/src/bri/zeroKnowledgeProof/api/dtos/request/createCCSMAnchor.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { IsNotEmpty } from 'class-validator';
+import { BpiSubjectAccount } from '../../../../identity/bpiSubjectAccounts/models/bpiSubjectAccount';
+import { BpiAccount } from '../../../../identity/bpiAccounts/models/bpiAccount';
+import { DocumentObject } from '../../../models/document';
+
+// TODO: Revisit validations rules once we move into business logic implementation
+export class CreateCCSMAnchorDto {
+  @IsNotEmpty()
+  ownerAccount: BpiSubjectAccount;
+
+  @IsNotEmpty()
+  agreementState: BpiAccount;
```

Comment:
probably another string for now, BpiAccount is something else . do we actually use this anywhere?